toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -541,8 +541,9 @@ build_rpm_in_chroot_no_install libxslt
 chroot_and_install_rpms pam
 build_rpm_in_chroot_no_install docbook-style-xsl
 
-# libsolv needs cmake
+# libsolv needs cmake, zstd-libs
```

Comment:
In the spec you've got a BR on `libzstd-devel`, which translates to `zstd-devel`, so I think either this comment or the spec is inaccurate.

Reply:
Thanks for the feedback. I've updated the comment and arguments for consistency.